nixos/modules/misc/locate.nix

```diff
@@ -188,24 +190,25 @@ in {
   };
 
   config = mkIf cfg.enable {
-    users.groups = mkIf isMLocate { mlocate = {}; };
+    users.groups = mkIf isMLocate { mlocate = {}; } // mkIf isPLocate { plocate = {}; };
 
-    security.wrappers = mkIf isMLocate {
+    security.wrappers = mkIf isMorPLocate {
       locate = {
-        group = "mlocate";
+        group = if isMLocate then "mlocate" else "plocate";
         owner = "root";
         permissions = "u+rx,g+x,o+x";
         setgid = true;
         setuid = false;
-        source = "${cfg.locate}/bin/locate";
+        source = "${cfg.locate}/bin/" + (if isMLocate then "locate" else "plocate");
       };
     };
 
+
     nixpkgs.config = { locate.dbfile = cfg.output; };
 
     environment.systemPackages = [ cfg.locate ];
 
-    environment.variables = mkIf (!isMLocate)
+    environment.variables = mkIf (!isMorPLocate)
       { LOCATE_PATH = cfg.output;
       };
```

Comment:
```suggestion
    environment.variables = mkIf (!isMorPLocate) {
      LOCATE_PATH = cfg.output;
    };
```

Reply:
I ended up running `nixfmt` on the entire file. I found multiple similar cases.
This is in a separate commit to clearly separate functional and stylistic changes.

Reply:
> I ended up running nixfmt on the entire file.

Please use nixpkgs-fmt since nixfmt has stupid ideas about line lengths.

Reply:
done